components/Templates/includes/functions-view_template.php

```diff
@@ -329,13 +329,25 @@ function frontier_do_subtemplate( $atts, $content ) {
 		} elseif ( 'file' == $field['type'] && 'attachment' == $field['options']['file_uploader'] ) {
 			$template = frontier_decode_template( $content, $atts );
 			foreach ( $entries as $key => $entry ) {
+				$entry_pod = pods( 'media', $entry['ID'] );
```

Comment:
Instead of setting up here, we should set up `$entry_pod` above, then check if it's `$entry_pod->valid()`  to determine if we'll handle the extra pod object work.

In the foreach, we can then `$entry_pod->fetch( $entry['ID'] )`

Reply:
```suggestion
```